contrib/hashicorp/vault/example_test.go

```diff
@@ -0,0 +1,95 @@
+package vault_test
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/contrib/hashicorp/vault"
```

Comment:
Note that we usually import the package name and rename it with the suffix of `*trace` in all our examples and use cases in documentation. I'm not sure if that's a good practice or not, but we usually do it because it tends to clash with the original package name. So here we would use:
```suggestion
	vaulttrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/hashicorp/vault"
```
Technically we should've had the `api` package but I think that's too generic so this should be good enough.